app/models/matching.rb

```diff
@@ -12,6 +12,24 @@
 #
 
 class Matching < ActiveRecord::Base
+
+  validates :instrument, presence: true
+  validates :lesson_time, presence: true, :length => { :minimum => 2}
+  validates :student_id, presence: true
+  validates :teacher_id, presence: true
+  validate :valid_lesson_array
+
   belongs_to :student, :inverse_of => :matchings
   belongs_to :teacher, :inverse_of => :matchings
+
+  def valid_lesson_array
+    if lesson_time.length > 1
+      lesson_time[0..-2].zip(lesson_time[1..-1]).each do |timestring1, timestring2|
+        unless (timestring2.to_i - timestring1.to_i)%24 == 1
+          errors.add(:lesson_time, "Times must be consecutive")
+        end
+      end
+    end
+  end
+
```

Comment:
this works but something else is fishy w the seed data that i'll look into tomorrow